jujugui/static/gui/src/app/jujulib/test-index.js

```diff
@@ -0,0 +1,39 @@
+/* Copyright (C) 2016 Canonical Ltd. */
+
+'use strict';
+
+chai.config.includeStack = true;
+chai.config.truncateThreshold = 0;
+
+describe('jujulib utility functions', function() {
```

Comment:
I'm sure if you checked a code coverage tool, the code would be thoroughly exercised via the various `test-jimm.js`, `test-charmstore.js`, etc. The problem was that those aren't proper unit tests of `_makeRequest`. Anyhow, glad we now have a home for direct, isolated testing of an important function.